support-frontend/app/controllers/Application.scala

```diff
@@ -126,26 +146,20 @@ class Application(
       paymentApiStripeEndpoint = paymentAPIService.stripeExecutePaymentEndpoint,
       paymentApiPayPalEndpoint = paymentAPIService.payPalCreatePaymentEndpoint,
       idUser = idUser,
-      stage = stage
+      preload = preload
     )
   }
 
   def showcase: Action[AnyContent] = CachedAction() { implicit request =>
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
     Ok(views.html.main(
       title = "Support the Guardian",
-      mainId = "showcase-landing-page",
-      mainJsBundle = "showcasePage.js",
-      mainStyleBundle = "showcasePage.css",
+      mainId = EmptyDiv("showcase-landing-page"),
+      mainJsBundle = RefPath("showcasePage.js"),
+      mainStyleBundle = Left(RefPath("showcasePage.css")),
```

Comment:
Left/Right Either is a little bit confusing, Left is the old way where we just link to the stylesheet, and Right is the StyleContent where we actually include it inline.
If this is less clear we can easily introduce a properly named ADT similar to what we have with EmptyDiv/SSRContent which might well be a lot clearer?  But I didn't want to go too trigger happy adding loads of descriptively named types in the same PR (I noticed everything is mostly String previously)